### PR TITLE
feat: install EasyMDE

### PR DESCRIPTION
**Pull request by PascalRepond**
Co-Authored-by: Pascal Repond <pascal.repond@rero.ch>

### CODE REVIEW

**Comment by PascalRepond** [comment]
Draft: implemented the EasyMDE editor. It works for existing pages but doesn't seem to work for new pages (`submit` button doesn't seem to do anything in flask and preview doesn't work). I couldn't locate the exact problem, need some help @jma

**Review by zannkukai — APPROVED** [review]

File: flask_wiki/static/js/wiki.js [hunk 1]
```diff
@@ -91,4 +91,12 @@ $(document).ready(function () {
...
     document.getElementById("confirm").href=url;
   });
+});
+
+// Load Easy MDE in editor-body
+const easyMDE = new EasyMDE({
+  spellChecker: false,
+  sideBySideFullscreen: false,
+  showIcons: ["code", "table", "heading-1", "heading-2", "heading-3"],
+  forceSync: true,
```
Comment (jma) [inline flask_wiki/static/js/wiki.js]: A small comment is welcome.

**Review by jma — APPROVED** [review]

File: flask_wiki/static/js/wiki.js (hunk 1 above)
Comment (PascalRepond) [reply flask_wiki/static/js/wiki.js]: Done

**Comment by PascalRepond** [comment]
Version was tested with rero-ils and sonar, will publish PRs on both projects when release `v0.2.0` is published.

**Pull request merged**